### DISCUSSION TITLE
Question about which proxies to buy

### DISCUSSION THREAD

**Discussion by 4leks1001** [KevinBytesTheDust/CVAmp#218] [Q&A] [answered]
Hello i just wanted to ask 4 things
1 Is there a difference between proxy servers and static residential
2 if there is which one would you recommend i buy if i just want to keep around 10 viewers
3 will it show up on my view count
4. [removed by Contributor]

**Comment by jlplenio** [comment, marked as answer]
Hello you.
1. Residential should be less detectable, but not needed for this tool. Save your money.
2. Follow this guide [Webshare.io Proxies Guide](https://github.com/jlplenio/crude-twitch-viewer-bot/wiki/Webshare.io-Proxies-Guide), when asked the _Proxy Amount_ select _Custom_ and choose 10 proxies.
3. Almost definitely.
4. We do not discuss this.
